library/lib_exec_node.ks

```diff
@@ -0,0 +1,43 @@
+@lazyglobal off.
+
+function MaxShipThrust{ 
+	// this function is needed as a workaround
+	// to kOS bug regarding ship thrust.
+	local engs is list().
+	list engines in engs.
+	local mth is 0.
+	for eng in engs{
+		if eng:VISP > 0 {
```

Comment:
This function was not written by me - it's taken from here: https://github.com/KSP-KOS/KOS/issues/940